src/app/compiler/compiler.js

```diff
@@ -261,15 +324,15 @@ function Compiler (handleImportCall) {
         if (err) {
           cb(err)
         } else {
-          files[m] = content
+          files[m] = { content }
           gatherImports(files, target, importHints, cb)
         }
       })
 
       return
     }
 
-    cb(null, { 'sources': files, 'target': target })
```

Comment:
I think `target` was needed in the compiler part and I don't see it removed above?